sql/parser/sql.y

```diff
@@ -3198,8 +3205,10 @@ array_expr_list:
   }
 
 extract_list:
-  extract_arg FROM a_expr {}
-| /* EMPTY */ {}
+  extract_arg FROM a_expr
+  {
+    $$ = Exprs{StrVal($1), $3}
+  }
 
 // Allow delimited string SCONST in extract_arg as an SQL extension.
 // - thomas 2001-04-12
```

Comment:
This is actually needed to support the use of a string as a parameter. 'year' instead of year